index.js

```diff
@@ -58,6 +61,9 @@ module.exports = {
 
     await nextOnNetlify({ functionsDir: FUNCTIONS_SRC, publishDir: PUBLISH_DIR })
   },
+  async onPostBuild({ inputs }) {
```

Comment:
We should add the new `inputs` to `manifest.yml`.

Reply:
RIGHT DUH